hsrc_lib/Database/LMDB/FFI.hsc

```diff
@@ -0,0 +1,293 @@
+{-# LANGUAGE CApiFFI #-}
+{-# LANGUAGE GeneralisedNewtypeDeriving #-}
+{-# LANGUAGE NamedFieldPuns #-}
+
+{-# OPTIONS_GHC -Wno-missing-export-lists #-}
```

Comment:
I think you could instead do `module Database.LMDB.FFI (..) where`, no?

Reply:
That didn't seem to work -- instead, I added a full export list